docs/CLI.md

```diff
@@ -61,3 +61,12 @@ gulp has very few flags to know about. All other flags are for tasks to use if n
 
 **--log-level**, **-L**
     Set the loglevel. -L for least verbose and -LLLL for most verbose. -LLL is default.
+
+**--config**, **-c**
+    Manually set path of configuration file.
+
+**--ignore-config**
+    Ignore configuration files except files specified by --config flag.
+
+**--output-config**
```

Comment:
That's an interesting use case.  I think each of these should come later and as separate PRs.